### PR TITLE
Handle single route for SPA

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #368

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@abdul-shajin _a Please check.
@vipulnsward Need your thoughts too on same.

**Comment by abdul-shajin** [comment]
LGTM :100:

**Comment by tejaswinichile** [comment]
@vipulnsward _a Please have a look.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def configure_permitted_parameters
 
     def after_sign_in_path_for(resource)
       if resource.organizations.exists?
-        organization_articles_path(resource.organizations.first.api_key)
+        "/organizations/#{resource.organizations.first.api_key}/articles"
```
Comment (vipulnsward) [inline app/controllers/application_controller.rb]: Please use path helper, no handcrafting of urls

File: config/routes.rb [hunk 2]
```diff
@@ -16,12 +16,9 @@
...
 
   resources :organizations, only: [:show], param: :api_key do
-    resources :articles
-    resources :urls
-    resources :categories
-    resources :tickets, only: [:index]
-    resources :feedbacks, only: [:index, :show]
-    resources :team, only: [:index]
+    get "*path", to: "admin/dashboard#index", constraints: -> (request) do
+      !request.xhr? && request.format.html?
```
Comment (vipulnsward) [inline config/routes.rb]: What happens if it indeed is an xhr? and html

**Comment by vipulnsward** [comment]
@breezyboa _a please take this up to see how it fits with FE routing

File: config/routes.rb (hunk 2 above)
Comment (tejaswinichile) [reply config/routes.rb]: An XHR request should always be handled by graphQL, not by this route.

**Comment by breezyboa** [comment]
This will handle all routes in `http://localhost:3000/organizations/*`via the frontend

To make this completely work we need all 404s also to be directed to a default page. it could be a root path like `http://localhost:3000/`. Let's take care of that separately

Merging this PR

**Pull request merged**